doc/modules/array_api.rst

```diff
@@ -9,7 +9,19 @@ Array API support (experimental)
 The `Array API <https://data-apis.org/array-api/latest/>`_ specification defines
 a standard API for all array manipulation libraries with a NumPy-like API.
 Scikit-learn's Array API support requires
-`array-api-compat <https://github.com/data-apis/array-api-compat>`__ to be installed.
+`array-api-compat <https://github.com/data-apis/array-api-compat>`__ to be installed,
+and the environment variable `SCIPY_ARRAY_API` must be set to `1` before importing
+`scipy` and `scikit-learn`:
+
+.. prompt:: bash $
+
+   export SCIPY_ARRAY_API=1
+
+Please note that this environment variable is intended for temporary use.
+For more details, refer to the following SciPy documentation:
+
+- https://docs.scipy.org/doc/scipy/dev/api-dev/array_api.html#using-array-api-standard-support
```

Comment:
> Note this is in principle a temporary thing in scipy while array API support is being worked on but I think this is worth mentioning anyway because this temporary thing may last a bit of time.

I found the same remark in SciPy's documentation. I hope adding the reference will be helpful.